packages/edgeidentity/__tests__/AEPEdgeIdentityTests.ts

```diff
@@ -29,4 +30,110 @@ describe('AEPEdgeIdentity', () => {
+
+    let identityItems1  = new AEPIdentityItem(identifier1, authenticatedState1, isPrimary1);
+    let identityItems2  = new AEPIdentityItem(identifier2, authenticatedState2, isPrimary2);
+  
+    let idmap = new AEPIdentityMap();
+
+    let expectedidmap = {"items": { "1stNameSpace" : [{"id": identifier1, "authenticatedState": authenticatedState1, "primary": isPrimary1}], "2ndNameSpace" : [{"id": identifier2, "authenticatedState": authenticatedState2, "primary": isPrimary2}]}};
+   
+
+    //add item 1
+    idmap.addItem(identityItems1, namespace1);
+
+    //add item 2
+    idmap.addItem(identityItems2, namespace2);
+    await AEPIdentity.updateIdentities(idmap);
+    expect(spy).toHaveBeenCalledWith(expectedidmap);
+  });
+
+ it('removeIdentity is called', async () => {
+    const spy = jest.spyOn(NativeModules.AEPEdgeIdentity, 'removeIdentity');
+    let identifier1 = "id1";
+    let namespace1 = "1stNameSpace"
+    let authenticatedState1 = AEPAuthenticatedState.AMBIGUOUS;
+    let isPrimary1 = true;
+
+    let identityItem1  = new AEPIdentityItem(identifier1, authenticatedState1, isPrimary1);
+
+    await AEPIdentity.removeIdentity(identityItem1, namespace1)
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('removeIdentityItem is validated', async () => {
+    const spy = jest.spyOn(NativeModules.AEPEdgeIdentity, 'updateIdentities');
+
+    let identifier1 = "id1";
+    let namespace1 = "1stNameSpace"
+    let authenticatedState1 = AEPAuthenticatedState.AMBIGUOUS;
+    let isPrimary1 = true;
+
+    let identifier2 = "id2";
+    let namespace2 = "2ndNameSpace"
+    let authenticatedState2 = AEPAuthenticatedState.AUTHENTICATED;
+    let isPrimary2 = false;
+
+    let identifier3 = "id3";
+    let authenticatedState3 = AEPAuthenticatedState.LOGGED_OUT;
+  
+    let identifier4 = "id4";
+
+    let identityItems1  = new AEPIdentityItem(identifier1, authenticatedState1, isPrimary1);
+    let identityItems2  = new AEPIdentityItem(identifier2, authenticatedState2, isPrimary2);
+    let identityItems3  = new AEPIdentityItem(identifier3, authenticatedState3, isPrimary2);
+    let identityItems4  = new AEPIdentityItem(identifier4, authenticatedState2);
+
+    let idmap1 = new AEPIdentityMap();
+
+        let expectedidmap = {"items": { "1stNameSpace" : [{"id": identifier1, "authenticatedState": authenticatedState1, "primary": isPrimary1}], "2ndNameSpace" : [{"id": identifier3, "authenticatedState": authenticatedState3, "primary": isPrimary2}, {"id": identifier4, "authenticatedState": authenticatedState2, "primary": isPrimary2}]}}; 
+
+    //add item 1
+    idmap1.addItem(identityItems1, namespace1);
+
+    //add item 2
+    idmap1.addItem(identityItems2, namespace1);
+
+    //add item 3
+    idmap1.addItem(identityItems3, namespace2);
+
+    //add item 4
+    idmap1.addItem(identityItems4, namespace2);
+
+    //remove item 2
+    idmap1.removeIdentityItem(identityItems2, namespace1);
+
+    let checkEmpty = idmap1.isEmpty();
```

Comment:
I am not sure how to validate the method in IdentityMap in jest.  I can only check the result of the methods under AEPIdentity.  So I just print out console.log to manually check right now.

Reply:
You can use expect(actual).toEqual(expected). fixed in this PR.